test/game_test.rb

```diff
@@ -0,0 +1,24 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/contestant'
+require "./lib/game"
+
+class GameTest < Minitest::Test
+  def test_it_exists
+    pick_4 = Game.new('Pick 4', 2)
+    mega_millions = Game.new('Mega Millions', 5, true)
+
+    assert_instance_of Game, pick_4
+    assert_instance_of Game, mega_millions
+  end
+
+  def test_it_has_attributes
+    mega_millions = Game.new('Mega Millions', 5, true)
+    pick_4 = Game.new('Pick 4', 2)
+
+    assert_equal "Mega Millions", mega_millions.name
+    assert_equal 5, mega_millions.cost
+    assert_equal true, mega_millions.national_drawing?
+    assert_equal false, pick_4.national_drawing?
```

Comment:
Similar to the out_of_state test, we also want to verify that this method _could_ return `true`